recipes/ftxui/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.11)
+project(test_package)
+
+SET(CMAKE_CXX_STANDARD 11)
```

Comment:
```suggestion
```

Reply:
Can you please explain? FTXUI definitely requires C++11.

Reply:
C++17 according to their CMakeLists

Reply:
So shouldn't we replace the 11 with 17 instead of just deleting it?

Reply:
`target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_17)` takes care of it.